pkg/scheduler/algorithmprovider/defaults/compatibility_test.go

```diff
@@ -612,52 +612,17 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
 
 	for v, tc := range schedulerFiles {
 		fmt.Printf("%s: Testing scheduler config\n", v)
```

Comment:
I think we don't need this printf anymore. (Also, they should have used (*T).Logf)